Makefile

```diff
@@ -22,18 +22,26 @@ check-dummy:
 		--bin=zeitgeist \
 		--features=parachain,try-runtime \
 		try-runtime \
-		--execution=Native \
```

Comment:
I think a good reason for Native execution is the ability to read errors instead of getting a wasm strapped message. Do we really need to use wasm execution to get try-runtime working?